packages/clarity-js/types/data.d.ts

```diff
@@ -92,7 +92,9 @@ export const enum Dimension {
     ProductCategory = 11,
     ProductSku = 12,
     ProductCurrency = 13,
-    ProductCondition = 14
+    ProductCondition = 14,
+    TabId = 15,
```

Comment:
Shouldn't this go to the envelope? Doesn't seem like it belongs here.

Reply:
Thought about it and decided to not include it in the envelope because it has no bearing on aggregation. It's a hint for visualization - if we choose to use it later.